app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,83 +1,165 @@
+            <%# end %>
           </li>
-        <% end %>
+        <%# end %>
       </ul>
-    <% end %>
+    <%# end %>
 
-    <% if is_product_available_in_currency && @product.can_supply? %>
-      <%= hidden_field_tag "variant_id", default_variant.id %>
+    <%# if is_product_available_in_currency && @product.can_supply? %>
+      <%#= hidden_field_tag "variant_id", default_variant.id %>
 
       <hr>
       <div>
         <div class="mb-5 mt-4 text-center text-md-left">
-          <%= render 'spree/shared/quantity_select', input_name: :quantity %>
+          <%#= render 'spree/shared/quantity_select', input_name: :quantity %>
         </div>
 
-        <%= button_tag class: 'btn btn-primary w-100 text-uppercase font-weight-bold add-to-cart-button', id: 'add-to-cart-button', type: :submit, disabled: true do %>
-          <%= Spree.t(:add_to_cart) %>
-        <% end %>
+        <%#= button_tag class: 'btn btn-primary w-100 text-uppercase font-weight-bold add-to-cart-button', id: 'add-to-cart-button', type: :submit, disabled: true do %>
+          <%#= Spree.t(:add_to_cart) %>
+        <%# end %>
       </div>
-    <% end %>
+    <%# end %>
+  </div> -->
+<%# end %>
+<% is_product_available_in_currency = product_available_in_currency? %>
+<% default_variant = default_variant(@variants, @product) %>
+<div class="product-side-details d-flex flex-column justify-content-between h-100"  >
+  <div class="top-detail" >
+    <%= form_for :order, html: {
+      id: 'add-to-cart-form',
+      class: 'add-to-cart-form',
+      'data-product-summary': @product_summary.to_json,
+      'data-variants': product_variants_matrix(is_product_available_in_currency),
+      'data-variant-change-trigger-identifier': variant_change_identifier
+      } do |f| %>
+        <div class="product-options-header d-flex align-items-center justify-content-between" id="inside-product-cart-form" data-hook="inside_product_cart_form">
+          <div class="left">Choose your <strong>option:</strong></div>
+          <div class="right">
+            <div class="text-center text-md-left text-uppercase " id="product_variant_sku">
+                <%= default_variant.sku  %>
+            </div>
+          </div>
+        </div>
+        <ul class="size-availability" id="product-variants">
+          <% if @product.variants_and_option_values(current_currency).any? %>
+            <% @product.variants_and_option_values(current_currency).each_with_index do |variant, index| %>  
+              <li class="d-flex flex-wrap align-items-center justify-content-between  product-ov-js <%= index == 0 ? 'active' : '' %>"
+                data-varient-id="<%= variant.id %>" data-varient-sku="<%= variant.sku %>"> 
+              <% option_value = variant.size_options %>
+                <div class="left  add-to-cart-form-price price selling"  id="product-price"  data-hook="product_price">
+                <%= variant.price_in(current_currency).money %><span>/ $6.29 per oz.</span></div>
```

Comment:
Please display the variant price using default spree price method: <%= display_price(default_variant) %>

pass variant object to this function only. It will display the price automatically.